src/test/java/org/datadog/jmxfetch/TestApp.java

```diff
@@ -494,7 +491,7 @@ public void testServiceCheckCounter() throws Exception {
         // Let's put a service check in the pipeline (we cannot call doIteration()
         // here unfortunately because it would call reportStatus which will flush
         // the count to the jmx_status.yaml file and reset the counter.
-        repo.sendServiceCheck("jmx", Status.STATUS_OK, "This is a test", "jmx_test_instance", null);
+        repo.sendServiceCheck("jmx", Status.STATUS_OK, "This is a test", null);
 
         // Let's check that the counter has been updated
         assertEquals(1, repo.getServiceCheckCount("jmx"));
```

Comment:
Oh that's right. 
Switching to JMX Remote is not worth. A better solution would be to be able to unit test the specific `getServiceCheckTags` method, and not instantiate the whole app like we are currently doing. This is what we do for instance with [TestConfiguration](https://github.com/DataDog/jmxfetch/blob/master/src/test/java/org/datadog/jmxfetch/TestConfiguration.java).
Let's see if it can be done easily and decide.